src/wizard/java/spring-boot.md

```diff
@@ -21,7 +21,7 @@ Install using either Maven or Gradle:
 </dependency>
 ```
 
-### Groovy
+### Gradle
 
 ```groovy {tabTitle:Gradle}
```

Comment:
maybe this should change then? to `Groovy`

Reply:
the programming language is groovy, so for syntax highlighting it's correct. But as a heading, we want to indicate this line is a "gradle installation" as opposed to Maven above (which is in XML form)